### PR TITLE
Support VirtIO disk

### PR DESCRIPTION
**Pull request by srbarrios**

### CODE REVIEW

File: kiwi-desc-saltboot/saltboot/suse-SLES12/config.xml [hunk 1]
```diff
@@ -155,6 +155,9 @@
         <file name="net/packet/*"/>
         <file name="net/sunrpc/*"/>
     </drivers>
+    <drivers> 
+        <file name="drivers/block/virtio_blk.ko" /> 
+    </drivers>
```
Comment (aaannz) [inline kiwi-desc-saltboot/saltboot/suse-SLES12/config.xml:160]: This should go up to the default profile section. There are already other block drivers.

**Comment by srbarrios** [comment]
@aaannz @nadvornik what I should do with this? Can I merge?

**Comment by aaannz** [comment]
Ah, sorry slipped my mind. With the profile I meant the another one `default,xen,wireless`. If you can just move it there. There are already list of block drivers.

**Comment by srbarrios** [comment]
@aaannz please merge when you approve, if not I will forget it :D Thanks!

**Review by aaannz — APPROVED** [review]
👍🏾

**Comment by aaannz** [comment]
I will add change log entry and merge.

**Pull request merged**